src/pretix/static/pretixpresale/scss/main.scss

```diff
@@ -1,3 +1,7 @@
+/* before variables.scss because it only affects presale, not control */
```

Comment:
/* before variables.scss as it affects presale only, not control */

Reply:
Hi @mariobehling, text is updated.